modules/ROOT/pages/_partials/api-governance.adoc

```diff
@@ -33,13 +33,29 @@ This command evaluates specified filters to determine the rulesets that would be
 
 Besides the default flags, this command also accepts:
 
-[cols="1,1"]
+[%header,cols="35a,65a"]
 |===
 |Flag |Description
 |`--api=<api>`
 |The API project against which you want to evaluate. The command uses the criteria in the project's `exchange.json` file.
-|`--criteria <filtertype:filtervalue>,<filtertype:filtervalue>,...`
-|Enables you to apply filters against which to evaluate to identify rulesets. Specify a list of comma-separated filters where each filter has a type and value in the format filterType:filterValue. Available filters include category, tag, status (lifecycle state), and scope (the targeted API types). Example: tag:tag1,category:category1:value,category:category2:value2,scope:async-api,status:stable,status:development,status:deprecated,scope:rest-api,scope:async-api
+|--criteria <filtertype:filtervalue>,...
+|Enables you to apply filters against which to evaluate to identify rulesets. Specify a list of comma-separated filters where each filter has a type and value in the format `filtertype:filtervalue`. 
+
+Available filters include:
+
+* `scope`: API type, `rest-api` or `async-api` 
+* `tag`: Tag defined for APIs in Exchange
+* `category`: Category defined for APIs in Exchange
```

Comment:
same comments in the previous pr, category could have an exaample, since it's "tricky" and the "APIs in the development lifecycle do not have instances" I prefer can not because they can't but I'll leave that up to you.